### ISSUE TITLE
sentora official merging

### ISSUE THREAD

**Issue by VedranIteh** [Dukecitysolutions/Sentora-Live-v1.0.3.1#1]
Hi @Dukecitysolutions ,
I used your updater to bump up to php7. I had quite a lot of broken features on CentOS 6 Sentora setup as the installer failed updating some stuff related to python 3 etc.... I would like to contribute fixes to Sentora in general and/or your project so I was wondering if you are planning to merge 1.0.3.1 with sentora core or continue a standalone development?

A straight up Sentora php7 installer would be nice for sentora project.

Best regards,
Vedran

**Comment by Dukecitysolutions** [comment]
Yes, I will. Sentora team can be very slow. I'm hoping with my v1.0.3.1 I can help move the project faster

**Comment by VedranIteh** [comment]
cool cool cool !
I'm gonna submit both ways but I have a feeling your effort could pull some new devs into this if only we could make them pull pr's faster and maybe merge your version.
